config/playground/helm/dev/helm-opensearch.yaml

```diff
@@ -0,0 +1,1005 @@
+                    # Skip users matching a user name, a wildcard or a regex pattern
+                    #skip_users:
+                    #  - 'cn=Michael Jackson,ou*people,o=TEST'
+                    #  - '/\S*/'
+              roles_from_another_ldap:
+                description: "Authorize via another Active Directory"
+                http_enabled: false
+                transport_enabled: false
+                authorization_backend:
+                  type: ldap
+                  #config goes here ...
+      internal_users.yml: |-
+        ---
+        # This is the internal user database
+        # The hash value is a bcrypt hash and can be generated with plugin/tools/hash.sh
+        
+        _meta:
+          type: "internalusers"
+          config_version: 2
+        
+        # Define your internal users here
+        
+        ## Demo users
+        
+        admin:
+          hash: "$2a$12$VcCDgh2NDk07JGN0rjGbM.Ad41qVR/YFJcgHp0UGns5JDymv..TOG"
+          reserved: true
+          backend_roles:
+          - "admin"
+          description: "Demo admin user"
+        
+        kibanaserver:
+          hash: "$2a$12$4AcgAt3xwOWadA5s5blL6ev39OXDNhmOesEoo33eZtrq2N0YrU3H."
+          reserved: true
+          description: "Demo OpenSearch Dashboards user"
+        
+        kibanaro:
+          hash: "$2a$12$JJSXNfTowz7Uu5ttXfeYpeYE0arACvcwlPBStB1F.MI7f0U9Z4DGC"
+          reserved: false
+          backend_roles:
+          - "kibanauser"
+          - "readall"
+          attributes:
+            attribute1: "value1"
+            attribute2: "value2"
+            attribute3: "value3"
+          description: "Demo OpenSearch Dashboards read only user"
+        
+        logstash:
+          hash: "$2a$12$u1ShR4l4uBS3Uv59Pa2y5.1uQuZBrZtmNfqB3iM/.jL0XoV9sghS2"
+          reserved: false
+          backend_roles:
+          - "logstash"
+          description: "Demo logstash user"
+        
+        readall:
+          hash: "$2a$12$ae4ycwzwvLtZxwZ82RmiEunBbIPiAmGZduBAjKN0TXdwQFtCwARz2"
+          reserved: false
+          backend_roles:
+          - "readall"
+          description: "Demo readall user"
+        
+        snapshotrestore:
+          hash: "$2y$12$DpwmetHKwgYnorbgdvORCenv4NAK8cPUg8AI6pxLCuWf/ALc0.v7W"
+          reserved: false
+          backend_roles:
+          - "snapshotrestore"
+          description: "Demo snapshotrestore user"
+      roles.yml: |-
+        _meta:
+          type: "roles"
+          config_version: 2
+        
+        # Restrict users so they can only view visualization and dashboard on OpenSearchDashboards
+        kibana_read_only:
+          reserved: true
+
+        opendistro_security_anonymous:
+          reserved: true
+          cluster_permissions:
+            # For using _cat/indices
+            - 'cluster:monitor/state'
+            - 'cluster:monitor/health'
+            # To enable read access to ISM
+            - cluster:admin/opendistro/ism/policy/search
```

Comment:
did you click each page? I didn't see snapshot Get list here?

Reply:
Done, snapshot permissions have been added.